.bazelrc

```diff
@@ -21,6 +21,14 @@ build --copt=-DGRPC_BAZEL_BUILD
 # inside a docker image.
 build --experimental_convenience_symlinks=ignore
 
+# Use host-OS-specific config lines from bazelrc files.
```

Comment:
AFAIK, this file is not loaded if `google-cloud-cpp` is built as part of a larger project.   That is not a blocker for this change, but maybe we should document the limitations somewhere?

Reply:
Added such a note.

[Indeed, if this code was part of a bigger project they may not be using `bazel` or `git`, so they'd have to figure out how to generate their own build identifier.]